astropy/utils/console.py

```diff
@@ -365,7 +365,7 @@ def strip_ansi_codes(s):
     """
     Remove ANSI color codes from the string.
     """
-    return re.sub('\033\[([0-9]+)(;[0-9]+)*m', '', s)
+    return re.sub(r'\033\[([0-9]+)(;[0-9]+)*m', '', s)
```

Comment:
The lesson may be that it does *not* pay to "make things easier for the user" -- it just ends being confusing.